pkg/executables/docker.go

```diff
@@ -70,3 +73,28 @@ func (d *Docker) AllocatedMemory(ctx context.Context) (uint64, error) {
 	totalMemory = totalMemory[1 : len(totalMemory)-2]
 	return strconv.ParseUint(totalMemory, 10, 64)
 }
+
+func (d *Docker) Pull(ctx context.Context, image string) error {
+	if _, err := d.executable.Execute(ctx, "pull", image); err != nil {
+		return err
+	}
+	return nil
+}
+
+func (d *Docker) TagImage(ctx context.Context, image string, endpoint string) error {
+	li := strings.ReplaceAll(image, defaultRegistry, endpoint)
```

Comment:
Maybe we should just call this variable `localImage` to make its purpose clear.

Reply:
done